src/utils/sources-tree.js

```diff
@@ -224,13 +225,27 @@ function addToTree(tree: any, source: TmpSource) {
 }
 
 /**
+ * @memberof utils/sources-tree
+ * @static
+ */
+function isExactUrlMatch(pathPart:string, debuggeeUrl:string) {
+  // compare to hostname with an optional 'www.' prefix
+  const urlParts = /^(?:\w+\:\/\/)?([^\/]+)(.*)$/.exec(debuggeeUrl);
+  if (urlParts == null) {
+    return false;
+  }
+  const baseUrl = urlParts[1].replace(/^www\./, "");
+  return pathPart === baseUrl || pathPart === `www.${baseUrl}`;
+}
```

Comment:
actually, i'm pretty sure there must be something in:

`URL.parse` or `new URL` that would give you a hostname you could compare